aries-site/src/themes/aries.js

```diff
@@ -6,12 +6,43 @@ export const aries = deepMerge(hpe, {
   // To be stripped out once theme changes are made in grommet-theme-hpe
   // keeping file for use as playground for future theme adjusments that need
   // to be quickly tested
-  menu: {
-    group: {
-      separator: {
-        color: 'border-weak',
+  feedback: {
+    body: {
+      gap: 'medium',
+    },
+    closeButton: {
+      a11yTitle: `You are in a layer containing a form to submit feedback.
```

Comment:
I'm not sure using the word 'layer' here makes sense to most users. At best they might understand 'dialog'